process/block/metablock.go

```diff
@@ -1175,7 +1186,30 @@ func (mp *metaProcessor) displayPoolsInfo() {
 	log.Debug("pools info",
 		"total headers", mp.dataPool.Headers().Len(),
 		"headers pool capacity", capacity,
+		"total miniblocks", mp.dataPool.MiniBlocks().Len(),
+		"miniblocks pool capacity", mp.dataPool.MiniBlocks().MaxSize(),
 	)
+
+	for _, hash := range mp.dataPool.MiniBlocks().Keys() {
+		value, ok := mp.dataPool.MiniBlocks().Get(hash)
+		if !ok {
+			log.Debug("displayPoolsInfo: mini block not found", "hash", logger.DisplayByteSlice(hash))
+			continue
+		}
+
+		miniBlock, ok := value.(*block.MiniBlock)
+		if !ok {
+			log.Debug("displayPoolsInfo: wrong type assertion to *block.MiniBlock")
+			continue
+		}
+
+		log.Debug("mini block from pool",
```

Comment:
log.trace

Reply:
Yep, this was set to debug only for the test purposes